pkg/reconciler/v1alpha1/clusteringress/clusteringress.go

```diff
@@ -53,6 +54,21 @@ type Reconciler struct {
 // Check that our Reconciler implements controller.Reconciler
 var _ controller.Reconciler = (*Reconciler)(nil)
 
+// TODO:greghaynes Move this in to pkg/controller as a general filter
+func filterAnnotation(key string, value string, allowUnset bool) func(interface{}) bool {
```

Comment:
This is useful!  cc @josephburnett 

Perhaps put it in `pkg/reconciler` in `knative/serving` and make public for now?